PixiEditor/Views/Dialogs/SettingsWindow.xaml

```diff
@@ -74,6 +76,15 @@
                     </StackPanel>
                 </StackPanel>
             </Grid>
+            <Grid Visibility="{Binding SelectedCategory, Converter={StaticResource EqualityBoolToVisibilityConverter},
+            ConverterParameter='Discord'}">
+                <StackPanel Orientation="Vertical">
+                    <Label Style="{StaticResource Header1}" Content="Rich Presence"/>
+                    <StackPanel Orientation="Vertical" Margin="50 0 50 0">
+                        <CheckBox IsChecked="{Binding SettingsSubViewModel.EnableRichPresence}" Content="Enable"/>
```

Comment:
Should be past tense, "Enabled", also it's good to add an option to disable particular parts, like Layers, Canvas size, name.